src/components/Questions.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import Answer from './Answers';
 
-export const Questions = (props) => {
-  
+const Questions = (props) => {
```

Comment:
I would rename this component and file to `Question` rather than `Questions`. It's only responsible for generating a single question, so it doesn't make a ton of sense to pluralize the name of the component.